include/ast/component/value_type.h

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2019-2022 Second State INC
+
+//=== wasmedge/ast/component/value_type.h - ValueType class
+// definitions
```

Comment:
I guess this is my formatter configuration problem, I would check how to solve this

Reply:
I try my neovim which uses `clang-format`, but still has this issue, should we config line width?
<img width="895" alt="圖片" src="https://user-images.githubusercontent.com/22004511/185827305-cdeef491-15ce-4c7e-83b2-f24030269bcf.png">

Reply:
I think some of the `-`s can be removed to adjust the line length.

Reply:
just tried
<img width="656" alt="圖片" src="https://user-images.githubusercontent.com/22004511/185854993-f1d7e494-2937-4d3d-846f-035c7a1e922f.png">

Reply:
Sad :-(.
Maybe we should also update the length limit in `.clang-format`. Is the default value `80`?

Reply:
Yes, updated.

Reply:
seems like this would also attack other code lol and make formatting check fail.

Reply:
Yup. Let's discuss the meeting to ensure we want to extend the column limit because it will affect all the code bases.

Let's accept the `newline` solution and make the `column limit` to the default value `80`.